src/Contracts/Models/WorkflowInstance.cs

```diff
@@ -0,0 +1,41 @@
+﻿// SPDX-FileCopyrightText: © 2021-2022 MONAI Consortium
+// SPDX-License-Identifier: Apache License 2.0
+
+using Microsoft.Extensions.Configuration;
+using Monai.Deploy.WorkflowManager.Contracts.Models;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace Monai.Deploy.WorkloadManager.Contracts.Models
+{
+    public class WorkflowInstance
+    {
+        [JsonIgnore]
+        public string Id { get; set; }
+
+        [ConfigurationKeyName("ae_title")]
```

Comment:
May want to use `JsonProperty` for models instead of `ConfigurationKeyName`

Reply:
Changed